features/media-regenerate.feature

```diff
@@ -12,13 +12,18 @@ Feature: Regenerate WordPress attachments
 
   Scenario: Delete existing thumbnails when media is regenerated
     Given download:
-      | path                        | url                                              |
-      | {CACHE_DIR}/large-image.jpg | http://wp-cli.org/behat-data/large-image.jpg     |
+      | path                              | url                                                   |
+      | {CACHE_DIR}/large-image.jpg       | http://wp-cli.org/behat-data/large-image.jpg          |
+      | {CACHE_DIR}/minimal-us-letter.pdf | http://localhost/behat-data/minimal-us-letter.pdf     |
```

Comment:
O cool, wot a system! Have done that https://github.com/wp-cli/wp-cli.github.com/pull/280